Initial commit of content

ralphjohnquiaoit opened the pull request.

ralphjohnquiaoit committed:
Initial commit of content

AgnesCorpuz commented on README.md:
Add a flowchart here

AgnesCorpuz commented on provider.tf:
```diff
@@ -0,0 +1,53 @@
+  aws_region   = "Enter your preferred region here"
+  access_token = "If you want to use this instead of the OAuth client ID and secret"
+
+  # Gateway Settings
```
Is there a need to include the Gateway and Proxy settings?

ralphjohnquiaoit committed:
Revision 01

- modified provider.tf
- modified the Setup Terraform section
- added flowchart
